__tests__/commands/_install.js

```diff
@@ -52,6 +52,7 @@ export async function getPackageVersion(config: Config, packagePath: string): Pr
 
 export async function run(
   factory: (config: Config, reporter: Reporter, lockfile: Lockfile) => Install,
+  flags: Object,
```

Comment:
No particular reason. I just wanted to make sure it was before the optional parameters.